### ISSUE TITLE
UCF-101 - Convert from avi to jpg - estimated run time

### ISSUE THREAD

**Issue by deema-A** [kenshohara/3D-ResNets-PyTorch#232]
python -m util_scripts.generate_video_jpgs avi_video_dir_path jpg_video_dir_path ucf101
This has taken me days, is it normal?
thanks

**Comment by guilhermesurek** [comment]
with activitynet it took me a week.

**Comment by titania7777** [comment]
i think that is not normal because in my case take a just few hours on my laptop

**Comment by Purav-Zumkhawala** [comment]
I agree with @titania7777 it took me only 3-4 hours to extract all the images from the UCF101 dataset. But I am using a GPU and that might be accelerating the program. UCF101 is a small dataset and it should not take days to extract the frames.

**Comment by paden118** [comment]
> I agree with @titania7777 it took me only 3-4 hours to extract all the images from the UCF101 dataset. But I am using a GPU and that might be accelerating the program. UCF101 is a small dataset and it should not take days to extract the frames.

I want to ask, I run the ucf101 Python script to convert JPG, but after running, only 46 directories have been converted. What's the matter?
